src/richie/apps/courses/migrations/0006_auto_20190807_2043.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.1.11 on 2019-08-07 18:43
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("courses", "0005_auto_20190717_0827")]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name="persontitletranslation", unique_together=set()
+        ),
+        migrations.RemoveField(model_name="persontitletranslation", name="master"),
+        migrations.RemoveField(model_name="person", name="first_name"),
+        migrations.RemoveField(model_name="person", name="last_name"),
+        migrations.RemoveField(model_name="person", name="person_title"),
```

Comment:
@2018D these modifications are not related to your PR so don't include them.
We are planning to remove these fields in the next BC release.